lib/elixir/test/elixir/code_test.exs

```diff
@@ -496,7 +496,13 @@ defmodule CodeTest do
   test "string_to_quoted returns error on incomplete escaped string" do
     assert Code.string_to_quoted("\"\\") ==
              {:error,
-              {[line: 1, column: 3], "missing terminator: \" (for string starting at line 1)", ""}}
+              {[
+                 line: 1,
+                 column: 1,
```

Comment:
I have my doubts about the correctness of this case here though

Reply:
It is correct according to our definition of MissingTerminator. But you need to add `end_column` here.

Reply:
Actually, maybe it should be swapped around. line and column always points to where the error happens and we have opening_line and opening_column to point to where it began.

Reply:
@lukaszsamson do you have any opinion on what the editor should show here? The line where the identifier starts or is it the line where it is not closed? (usually the last line of the file)?

Reply:
I was also thinking about this, currently on the diagnostics that are printed, the location is shown as the file bottom, but it's not a problem there because the new snippet shows both the start and end

For a language server or editor extension that needs to show one point, I think it makes sense for us to use the starting one in the unclosed delimiter case, as the problem is always there and the end is always empty. For the mismatched delimiter I'm not sure

Reply:
> But you need to add end_column here

Currently our TokenMissingError does not support the end_column, we actually just calculate them when building the snippet

Should we update it so that we calculate the end column based on the snippet before building the exception? And if so, should we do it on this PR or another one?

Reply:
I have created this issue to help with the discussion: https://github.com/elixir-lang/elixir/issues/13187